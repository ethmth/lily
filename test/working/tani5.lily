let sorted_list::int = [45, 22, 11, 89, 76, 54]
sorted_list.selectionsort()
print(sorted_list)