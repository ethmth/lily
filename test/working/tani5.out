[11, 22, 45, 54, 76, 89]